open-media-match/src/OpenMediaMatch/__init__.py

```diff
@@ -5,9 +5,7 @@
 import flask_migrate
```

Comment:
(Unrelated to diff): I'm always suspicious of usage of `__init__` (it's very easy to accidentally create problems in `foo.__init__` that trigger when doing `import foo.bar`, and I'm doubly suspicious of files that do work on import.

For example, we may find it hard to write unittests on files under OpenMediaMatch because all these lines are being executed each test.

Is there is an equivalent of 
```
if __name__ == '__main__':
```
for flask apps?

Reply:
This is a great point and I spent far too long yesterday fighting circular import problems. I don't know the answer but I'll research a bit more. I'm not a Flask expert 😅